src/sentry/receivers/releases.py

```diff
@@ -104,13 +106,14 @@ def resolved_in_commit(instance, created, **kwargs):
     self_assign_issue: str = "0"
     if user_list:
         acting_user = user_list[0]
-        self_assign_issue = get_option_from_list(
-            user_option_service.get_many(
-                filter={"user_ids": [acting_user.id], "keys": ["self_assign_issue"]}
-            ),
-            key="self_assign_issue",
-            default="0",
-        )
+        with in_test_hide_transaction_boundary():
```

Comment:
Ahh okay, so I thought this represented a potential inefficiency/source of transaction timeouts since it still crosses the RPC boundary so I figured this was still something we wanted to flag with this decorator.

If we want to greenlight read-only RPC calls that's fine, but I do think there's value in keeping track of this kind of call.

Reply:
I agree, I actually prefer requiring the decorator even on reads.  They represent potential future improvements because honestly RPC inside of a transaction is a fragile design and it will cause us issues.

In this case, I believe we can introduce some replication to make this rpc interaction go away in the future.